apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -1317,6 +1324,20 @@ export const extensions: Extensions = {
                             "application that should not be secured, and do not require authentication. Multiple " +
                             "URIs can be set using <3>comma separated</3> values."
                     }
+                },
+                m2m: {
+                    configurations: {
+                        clientId: {
+                            hint: "The Client ID generated when registering the application in Asgardeo."
+                        },
+                        clientSecret: {
+                            hint: "The Client Secret generated when registering the application in Asgardeo."
+                        },
+                        tokenRequest: "The API request used by the application to obtain an access token from Asgardeo.",
+                        tokenEndpoint: "The token endpoint of Asgardeo.",
+                        scopes: "The list of authorized scopes." +
+                                "Include required scopes by authorizing APIs through the <1>API Authorization</1> tab"
```

Comment:
WDYT?

```suggestion
                                "Include additional required scopes by authorizing APIs through the <1>API Authorization</1> tab"
```

Reply:
IMO it's better to proceed without the `additional` word as sometimes there will be no authorized scopes for the newly created application.